### PR TITLE
Changes to support AMI builds with packer

### PR DESCRIPTION
**Pull request by ukanga**
- Add pgpass configuration for database connections w/o passwords
- Add sentry integration
- Configurable superset_webserver_timeout
- Changes to support AMI packer builds.

### CODE REVIEW

**Comment by nateMJ** [comment]
When adding new pgpass configs, will it overwrite the existing pgpass file or append to it?

**Comment by ukanga** [comment]
@nateMJ It will overwrite. The expectation is that the `.pgpass` details are maintained in your inventories and not on the server directly. This has been addressed accordingly for our setup.

File: templates/uwsgi.ini.j2 [hunk 1]
```diff
@@ -22,7 +22,9 @@ static-map={{ superset_wsgi_static_map }}
 {% endif %}
 buffer-size={{ superset_wsgi_buffer_size }}
 reload-on-rss={{ superset_reload_on_rss }}
+{% if superset_version is version_compare('0.35.0', '>=') %}
```
Comment (moshthepitt) [inline templates/uwsgi.ini.j2]: Is this only for `0.35.0`?  Curious coz we haven't deployed that version afaik

Comment (ukanga) [reply templates/uwsgi.ini.j2]: @moshthepitt Yes, the [create_app()](https://github.com/apache/incubator-superset/blame/a9ff51bd6133559c71a43318f46c80e0fe913618/superset/app.py#L50) function  was just introduced recently. Which is what we would normally use `superset.app:create_app()`.

**Pull request merged**